Src/Couchbase.Linq.IntegrationTests/BucketContextTests.cs

```diff
@@ -367,6 +367,26 @@ public void SubmitChanges_WhenDocsAreModifiedAndEndChangeTracking_DocumentIsNotM
             var doc = ClusterHelper.GetBucket("beer-sample").GetDocument<Beer>(((ITrackedDocumentNode)beer).Metadata.Id);
             Assert.AreNotEqual(beer.Abv, doc.Content.Abv);
         }
+
+        [Test]
+        public void Underlying_Bucket_can_be_used_to_execute_N1QL_query_directly()
+        {
+            // setup bucket context
+            var db = new BucketContext(ClusterHelper.GetBucket("beer-sample"));
+
+            // get a beer from the db
+            var beer = db.Query<Beer>().Select(b => N1QlFunctions.Meta(b)).First();
+
+            // execute N1QL to update the beer
+            var newBeerName = Guid.NewGuid().ToString();
+            var n1ql = string.Format("UPDATE `beer-sample` USE KEYS '{0}' SET name = '{1}';", beer.Id, newBeerName);
+            db.Bucket.Query<int>(n1ql);
```

Comment:
Yes, I didn't emote that but that is a very big advantage of UPDATE which you cannot do easily through K/V.

-Jeff